packages/ra-core/src/CoreAdminRouter.tsx

```diff
@@ -78,7 +78,9 @@ export class CoreAdminRouter extends Component<
     ) => {
         const { authProvider } = props;
         try {
-            const permissions = await authProvider(AUTH_GET_PERMISSIONS);
+            const permissions = authProvider
+                ? await authProvider(AUTH_GET_PERMISSIONS)
+                : null;
```

Comment:
I'll wait for more people to comment but I think `undefined` would make more sense here

Reply:
More détails: null could mean no permissions whereas undefined would mean, we didn't even checked

Reply:
So should I change it to undefined?

Reply:
Just wanted to clarify :) I'd still like to wait for more people to agree or disagree

Reply:
Let's go for `undefined`